metaflow/plugins/kubernetes/kuberenetes_utils.py

```diff
@@ -97,7 +97,13 @@ def get_deserialized_object(self):
                 "memory": "%sM" % str(args["memory"]),
                 "ephemeral-storage": "%sM" % str(args["disk"]),
             },
-            limits=compute_resource_limits(args),
+            limits={
+                **compute_resource_limits(args),
+                **{
+                "cpu": str(args["cpu"]),
+                "memory": "%sM" % str(args["memory"]),
+                }
+            }
```

Comment:
You can already do @kubernetes(resource_limits_memory="200M" resource_limits_cpu=2)   inside a kubernetes decorator. See the compute_resource_limits function.

With your change applied, any random Metaflow user who requests CPU 10 and Memory 2028 will automatically have that change applied their manifest, even if they didn't want it. So this would be considered a possible breaking change to Metaflow and shouldn't be in any public PR.

Reply:
I don't think so
the cpu/memory come from kubernetes decorator.  This is isolated.  Then all this does it uses for limits, if you look above it its the same code for requests.